docs/usage.rst

```diff
@@ -2,16 +2,123 @@
-2. Call ``conda env update --prune --file environment.yml``
+.. code-block:: yaml
+
+    {% set conda_py = os.environ.get('CONDA_PY', '35') %}
+    name: web-ui-py{{ conda_py }}
+
+    dependencies:
+      {% if sys.platform.startswith('linux') %}
+      - gcc
+      {% endif %}
+
+
+The following variables are available in the Jira 2 namespace:
+
+* ``root``: this is the full path to the directory containing the ``environment.devenv.yml`` file;
+* ``os``: standard module;
+* ``sys``: standard module;
+* ``platform``: standard module;
+
+
+Environment Variables
+=====================
+
+It is possible to define environment variables that should be configured in the environment when activated.
+
+.. code-block:: yaml
+
+    environment:
+      PATH:
+        - {{ root }}/bin
+      PYTHONPATH:
+        - {{ root }}/source/python
+      DB_LOCATION: https://localhost/dev
+
+Environment variables defined in *list form* (like ``PATH`` and ``PYTHONPATH`` above) will **append** to existing
+variables, using the appropriate separator for the platform (``:`` on Linux/OSX and ``;`` on Windows).
+
+Environment variables defined as a single string (like ``DB_LOCATION`` above) will **overwrite** an existing
+variable with the same name.
+
+``conda-devenv`` restores the variables of the environment to their original state upon deactivation.
+
+Includes
+========
+
+It is possible to use *include* directives to include one or more ``environment.devenv.yml`` files. This merges all
+``dependencies`` and ``environment`` definitions into a single environment, which makes it a good solution to work
+in one or more repositories in development mode.
+
+For example:
+
+``/home/user/projects/core/environment.devenv.yml``:
+
+.. code-block:: yaml
+
+    name: core
+    dependencies:
+      - numpy
+      - pandas
+      - pytest
+      - invoke
+    environment:
+      PYTHONPATH:
+        - {{ root }}/source/python
+
+
+``/home/user/projects/web-ui/environment.devenv.yml``:
+
+.. code-block:: yaml
+
+    name: web-ui
+    includes:
+      - {{ root }}/../core/environment.devenv.yml
+    dependencies:
+      - flask
+      - jinja2
+    environment:
+      PYTHONPATH:
+        - {{ root }}/source/python
+      PATH:
+        - {{ root }}/bin
+
+In this setup, all the user has to do is executing ``conda devenv``:
+
+.. code-block:: console
+
+    $ cd ~/projects/web-ui
+    $ conda devenv
+
+This will create a ``conda`` environment named ``web-ui`` containing all the dependencies and environment variables
+defined in both files.
+
+How it works
+============
+
+Here's how ``conda-devenv`` works behind the scenes:
+
+1. Generate an ``environment.yml`` file in the same directory as the ``environment.devenv.yml`` file. The generated
+   ``environment.yml`` should **not** be added to VCS.
+2. Call ``conda env update --prune --file environment.yml``.
+3. Generate ``devenv-activate{.sh,.bat}`` and ``devenv-deactivate{.sh,.bat}`` scripts in ``$PREFIX/etc/conda/activate.d``
```

Comment:
.fish is missing from the list

Also it may be worth listing somewhere the supported shells. If people start to use ```conda-devenv``` this question will certainly come up in the future, specially as conda introduces support for new shells (e.g. powershell)

The current version should work with bash, zsh, fish and cmd